source/fundamentals/linq.txt

```diff
@@ -0,0 +1,531 @@
+    var queryableCollection = _restaurantsCollection.AsQueryable();
+
+Once the collection is queryable, you can compose a query using 
+**query comprehension syntax** or **method syntax**.
+
+Select the :guilabel:`Query Syntax` or :guilabel:`Method Syntax` tab to see how 
+how to compose a query using LINQ:
+
+.. tabs::
+
+   .. tab:: Query Syntax
+      :tabid: query-syntax
+
+      .. code-block:: csharp
+
+         var query = from r in queryableCollection
+             where r.Name == "The Movable Feast"
+             select new { r.Name, r.Address };
+
+   .. tab:: Method Syntax
+      :tabid: method-syntax
+
+      .. code-block:: csharp
+
+         var query = queryableCollection
+             .Where(r => r.Name == "The Movable Feast")
+             .Select(r => new { r.Name, r.Address });
+
+The documents found in the query are returned as an object that implements the
+`IQueryable <https://learn.microsoft.com/en-us/dotnet/api/system.linq.iqueryable-1?redirectedfrom=MSDN&view=net-6.0>`__
+interface. You can print the results of the preceding example as follows: 
+
+.. io-code-block::
+
+   .. input::
+      :language: csharp
+
+      foreach (var restaurant in query)
+      {
+          Console.WriteLine(restaurant.ToJson());
+      }
+
+   .. output::
+      
+      { "name" : "The Movable Feast", "address" : { "building" : "284", "coord" : [-73.982923900000003, 40.6580753], "street" : "Prospect Park West", "zipcode" : "11215" } }
+
+Supported Aggregation Stages
+----------------------------
+
+You can create an aggregation pipeline by using LINQ with supported aggregation
+operations. To learn more about each aggregation pipeline stage, see the server
+manual :ref:`aggregation-pipeline-operator-reference` page.
+
+$project
+~~~~~~~~
+
+The ``$project`` aggregation stage returns a document using only the specified
+fields.
+
+Select the :guilabel:`Query Syntax` or :guilabel:`Method Syntax` tab to see how 
+how to generate a ``$project`` stage using LINQ:
+
+.. tabs::
+
+   .. tab:: Query Syntax
+      :tabid: query-syntax
+
+      .. code-block:: csharp
+         :emphasize-lines: 2
+
+         var query = from r in queryableCollection
+             select new { r.Name, r.Address };
+
+   .. tab:: Method Syntax
+      :tabid: method-syntax
+
+      .. code-block:: csharp
+         :emphasize-lines: 2
+
+         var query = queryableCollection
+             .Select(r => new { r.Name, r.Address });
+
+The preceding example returns the following document:
+
+.. code-block:: json
+
+   { "name" : "The Movable Feast", "address" : { "building" : "284", "coord" : [-73.982923900000003, 40.6580753], "street" : "Prospect Park West", "zipcode" : "11215" } }
+
+.. note:: Excluding the ``_id`` Field
+
+   MongoDB includes the ``_id`` field in all projections unless explicitly excluded. 
+   The {+driver-short+} automatically adds this exclusion when using LINQ to 
+   create a projection without ``_id``.
```

Comment:
I tried to clarify it a little better, let me know if it helps. Basically it's saying you don't need to explicitly exclude _id when you are doing LINQ queries like you do when you are directly querying MongoDB through the server

Reply:
I think I get it. What's tripping me up is the phrase "adds this exclusion," because it could mean either that the driver excludes the _id or that it includes it (i.e., "adds it back in"). 

Is this enough?

   If you don't include the ``_id`` field in your LINQ projection, 
   the {+driver-short+} excludes it from the results.